packages/relayer/processor/process_message.go

```diff
@@ -461,3 +452,55 @@ func (p *Processor) saveMessageStatusChangedEvent(
 
 	return nil
 }
+
+func (p *Processor) setGasTipOrPrice(ctx context.Context, auth *bind.TransactOpts) error {
+	gasTipCap, err := p.destEthClient.SuggestGasTipCap(ctx)
+	if err != nil {
+		if IsMaxPriorityFeePerGasNotFoundError(err) {
+			auth.GasTipCap = FallbackGasTipCap
+		} else {
+			gasPrice, err := p.destEthClient.SuggestGasPrice(context.Background())
+			if err != nil {
+				return errors.Wrap(err, "p.destBridge.SuggestGasPrice")
+			}
+			auth.GasPrice = gasPrice
+		}
+	} else {
+		auth.GasTipCap = gasTipCap
+
+		return nil
```

Comment:
this was marked as resolved but i dont think it was fixed, i think this `return nil` is extraneous, 

i think thi sfunction can be:

```go
func (p *Processor) setGasTipOrPrice(ctx context.Context, auth *bind.TransactOpts) error {
	gasTipCap, err := p.destEthClient.SuggestGasTipCap(ctx)
	if err != nil {
		if IsMaxPriorityFeePerGasNotFoundError(err) {
			auth.GasTipCap = FallbackGasTipCap
		} else {
			gasPrice, err := p.destEthClient.SuggestGasPrice(context.Background())
			if err != nil {
				return errors.Wrap(err, "p.destBridge.SuggestGasPrice")
			}
			auth.GasPrice = gasPrice
		}
		return nil
	} 

	auth.GasTipCap = gasTipCap

	return nil
```

eliminating the else clause completely